app/views/events/index.html.erb

```diff
@@ -2,10 +2,15 @@
 <%= link_to 'Add your Event', new_event_path %><br>
 <br>
 
-<% if current_user %>
-  <%= form_with url: '/users/sign_out', method: "delete", local: true do |form| %>
-    <%= form.submit value: "Log out" %>
-  <% end %>
-<% else %>
- <%= link_to 'Log in', new_user_session_path %>
+<% @events.reverse.each do |event| %>
+  <%= event.title %>
```

Comment:
Shall we have 'Event Title:' etc displaying before the name of event, and for the other fields too?